profiling/src/main.rs

```diff
@@ -14,11 +14,16 @@
 //! callgrind_annotate --auto=yes callgrind.out.35583 >out.rs
 //! qcachegrind callgrind.out.35583
 //! ```
+extern crate structopt;
```

Comment:
Same for that addition of extern-crate. Am I right that the changes in this file were added by accident? If not, maybe another `git pull origin master` might help.